images/krte/create_bazel_cache_rcs.sh

```diff
@@ -37,6 +37,10 @@ command_to_package () {
 
 # get the installed package version relating to a binary
 command_to_version () {
+    # skip commands that aren't installed
```

Comment:
is ok to skip and not to fail?

Reply:
there's no version if the command isn't installed.
this ensures that if we _do_ install the command later we'll record the version, but until then we record ``

Reply:
in this case rpmbuild isn't installed because this image currently only supports what kind builds, as opposed to `kubekins-e2e` which supports the kitchen sink.

if we were to `bazel build //...` in kubernetes we'd need `rpmbuild`.